worker/transfer.go

```diff
@@ -242,6 +253,7 @@ func parallelDownloadSlab(ctx context.Context, ss object.SlabSlice, hosts []sect
 				timer := time.NewTimer(downloadSectorTimeout)
 				select {
 				case <-timer.C:
+					span.SetAttributes(attribute.Bool("slow", true))
```

Comment:
`nit` wondering whether a `string` attribute passing the `downloadSectorTimeout` is slightly better since it's configurable - fine to leave as is if you want though

Reply:
meh, kind of changed my mind already, will resolve and merge 👍